charts/langsmith/templates/backend/secrets.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if not .Values.config.basicAuth.existingSecretsName }}
+
+{{- $secretName := include "langsmith.basicAuthSecretsName" . }}
+{{- $secretObj := (lookup "v1" "Secret" .Release.Namespace $secretName) | default dict }}
+{{- $secretData := (get $secretObj "data") | default dict }}
+
+{{- $jwtSecretFromValues := .Values.config.basicAuth.jwtSecret | default "" }}
+{{- $jwtSecretFromSecret := (get $secretData "jwt_secret") | default "" }}
+{{- $jwtSecret := ternary $jwtSecretFromSecret (b64enc $jwtSecretFromValues)  (empty $jwtSecretFromValues) }}
```

Comment:
I was actually thinking we should autogenerate if values are empty and no existing secret name is given - wdyt? https://blog.cloudcover.ch/posts/reusing-existing-kubernetes-secrets-in-helm-templates/

Reply:
So i actually did this for a previous chart but the issue is this fails with argo/helm template(for people who don't use helm cli directly) since the lookup will always fail

Reply:
Ok I'll update 👍